opencensus/stats/view_data.h

```diff
@@ -69,7 +69,13 @@ class ViewData {
   const DataMap<int64_t>& int_data() const;
   const DataMap<Distribution>& distribution_data() const;
 
+  // returns a start time for each timeseries/tag map.
```

Comment:
"Returns a start time for the view data."

Shouldn't this be marked deprecated similarly to the view_impl changes? This file is the actual public-facing API.

Reply:
Done and Done